lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -96,6 +97,26 @@ def ask_to_delete_backup_after_restore
         end
       end
 
+      def ask_for_tables_to_exclude_in_dump
+        if action == :dump && should_exlcude_tables?
```

Comment:
typo: should_exlcude_tables -> should_exclude_tables

+2 points for making the typo consistently in the method name.  😉 

Don't worry, I type `destory` instead of `destroy`.

Reply:
`performance` is the bane of my existence usually... ends up as `preformance`...

Good catch.  Thanks!

Reply:
> +2 points for making the typo consistently in the method name. 😉

Autocomplete...